server/views/pages/contacts/manage/contactSearch.njk

```diff
@@ -0,0 +1,131 @@
+            href: DPS_HOME_PAGE_URL
+          },
+          {
+              text: "Contacts",
+              href: "/"
+          },
+          {
+            text: prisonerDetails.lastName | convertToTitleCase + ", " + prisonerDetails.firstName | convertToTitleCase,
+            href: prisonerResultsLink
+        }
+      ]
+      }) }}
+  </div>
+</div>
+
+{{ miniProfile(prisonerDetails, '/prisoner-image/' + prisonerDetails.prisonerNumber, digitalPrisonServicesUrl + '/prisoner/' + prisonerDetails.prisonerNumber) }}
+
+{% include '../../../partials/formErrorSummary.njk' %}
+<span class="govuk-caption-l">Manage Contacts</span>
+<h1 class="govuk-heading-l">{{ title }}</h1>
+ <div class="govuk-grid-row">
+        <div class="govuk-grid-column-three-quarters">
+            <form method='POST'>
+                <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
+
+                {{ govukInput({
+                    id: "firstName",
+                    name: "firstName",
+                    label: { text: "First name" },
+                    classes: 'govuk-!-width-one-third',
+                    errorMessage: validationErrors | findError('firstName'),
+                    value: view.firstName
+                }) }}
+
+                {{ govukInput({
+                    id: "middleName",
+                    name: "middleName",
+                    label: { text: "Middle names" },
+                    classes: 'govuk-!-width-one-third',
+                    errorMessage: validationErrors | findError('middleName'),
+                    value: view.middleName
+                }) }}
+
+                {{ govukInput({
+                    id: "lastName",
+                    name: "lastName",
+                    label: { text: "Last name" },
+                    classes: 'govuk-!-width-one-third',
+                    errorMessage: validationErrors | findError('lastName'),
+                    value: view.lastName
+                }) }}
+
+                {% if validationErrors %}
+                    <span class="govuk-error-message">{{ validationErrors['day'] }}</span>
+                    <span class="govuk-error-message">{{ validationErrors['month'] }}</span>
+                    <span class="govuk-error-message">{{ validationErrors['year'] }}</span>
+                    <span class="govuk-error-message">{{ validationErrors['dob'] }}</span>
+                {% endif %}
+                {{ govukDateInput({
+                    id: "dob",
+                    name: "dob",
+                    fieldset: {
+                        legend: {
+                            text: "Date of birth"
+                        }
+                    },
+                    hint: {
+                        text: "For example, 15 6 2000"
+                    },
+                    errorMessage: {
+                        test: ''
+                    } if validationErrors and not validationErrors['lastName'],
```

Comment:
What is this doing?

Reply:
I have added to display the red border, but I have refactored to have the error messages just above the fields (day,month,year) assigned to the errorMessage property

Reply:
Ah I see